fdbclient/ThreadSafeTransaction.actor.cpp

```diff
@@ -157,6 +157,16 @@ ThreadFuture< Key > ThreadSafeTransaction::getKey( const KeySelectorRef& key, bo
 		} );
 }
 
+ThreadFuture<int64_t> ThreadSafeTransaction::getEstimatedRangeSizeBytes( const KeyRangeRef& keys ) {
+	KeyRange r = keys;
+
+	ReadYourWritesTransaction *tr = this->tr;
+	return onMainThread( [tr, r]() -> Future<int64_t> {
+			return tr->getEstimatedRangeSizeBytes(r);
```

Comment:
We should check check deferred errors here before calling the underlying function (see `getKey` right above for an example).

Reply:
Yes. I don't know how I missed that line from the beginning lol.